lib/notifications/display.dart

```diff
@@ -103,8 +126,15 @@ class NotificationDisplayManager {
       FcmMessageDmRecipient() =>
         data.senderFullName,
     };
-    final groupKey = _groupKey(data);
-    final conversationKey = _conversationKey(data, groupKey);
+
+    messagingStyle.messages?.add(MessagingStyleMessage(
```

Comment:
```suggestion
    messagingStyle.messages!.add(MessagingStyleMessage(
```

If `messages` here were somehow null, it'd be wrong for us to skip adding this new message.

So if it actually could be null, this should be like `(messagingStyle.messages ??= []).add(…`. But I think in fact it can't.

That also might be a sign that the wrapper API should just take a list here, rather than a nullable list. There's no distinction being expressed by null vs. an empty list, right?